urdf/inc/ur_common.xacro

```diff
@@ -41,14 +41,15 @@
     <mesh filename="${mesh}"/>
   </xacro:macro>
 
-  <xacro:macro name="read_model_data" params="joint_limits_parameters_file kinematics_parameters_file physical_parameters_file visual_parameters_file force_abs_paths">
+  <xacro:macro name="read_model_data" params="joint_limits_parameters_file kinematics_parameters_file physical_parameters_file visual_parameters_file joint_dynamics force_abs_paths">
```

Comment:
I think I would prefer passing the filename down to here as we do with the other parameters, as well.

Reply:
Actually, I am not sure. It would be nicer to keep it on one level, however this way it is backwards compatible.